<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabsWidget_personComputer">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Persons">
       <attribute name="title">
        <string>Persons</string>
       </attribute>
       <widget class="QLineEdit" name="searchInput_Person">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>461</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string> Search persons by name...</string>
        </property>
       </widget>
       <widget class="QTableView" name="table_Person">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>601</width>
          <height>311</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QWidget" name="widget_moreFilterOpsPersons" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>601</width>
          <height>91</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_gender">
         <property name="geometry">
          <rect>
           <x>19</x>
           <y>0</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Gender:</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_female">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Female</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_male">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Male</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_diedSearch">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>50</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Died in the years:</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>93</x>
           <y>10</y>
           <width>20</width>
           <height>71</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="input_searchDiedFrom">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>50</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string> From</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_bornSearch">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Born in the years:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="input_searchDiedTo">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>50</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string> To</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="input_searchBornTo">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>10</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string> To</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="input_searchBornFrom">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>10</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string> From</string>
         </property>
        </widget>
        <widget class="QComboBox" name="input_searchNat">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>40</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
        <widget class="QLabel" name="label_nat">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>10</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Nationality:</string>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>10</y>
           <width>20</width>
           <height>71</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>601</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="button_advSearchPerson">
        <property name="geometry">
         <rect>
          <x>472</x>
          <y>10</y>
          <width>131</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Advanced search</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_computers">
       <attribute name="title">
        <string>Computers</string>
       </attribute>
       <widget class="QComboBox" name="comboBox_searchComputer">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>104</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLineEdit" name="searchInput_Comp">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>491</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string> Search computers by name...</string>
        </property>
       </widget>
       <widget class="QTableView" name="table_Comp">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>601</width>
          <height>411</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuComputer_History">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_new_person"/>
    <addaction name="actionAdd_new_computer"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit_person"/>
    <addaction name="actionEdit_Computer"/>
   </widget>
   <addaction name="menuComputer_History"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_new_person">
   <property name="text">
    <string>Add new person</string>
   </property>
  </action>
  <action name="actionAdd_new_computer">
   <property name="text">
    <string>Add new computer</string>
   </property>
  </action>
  <action name="actionEdit_person">
   <property name="text">
    <string>Edit person</string>
   </property>
  </action>
  <action name="actionEdit_Computer">
   <property name="text">
    <string>Edit Computer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
